packages/deploy/deploy/500_marketplace/501_royalty_registry_set_ownership.ts

```diff
@@ -0,0 +1,24 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+const func: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+): Promise<void> {
+  const {deployments, getNamedAccounts} = hre;
+  const {execute, read, catchUnknownSigner} = deployments;
+  const {deployer, sandAdmin} = await getNamedAccounts();
+  if (!((await read('RoyaltiesRegistry', 'owner')) == sandAdmin)) {
```

Comment:
not a big fan of inline await (its done like this in asset but still)
i'd rather have
```
owner = await read('RoyaltiesRegistry', 'owner'));
if (!((owner == sandAdmin)) {
```